tldw_Server_API/app/core/DB_Management/content_backend.py

```
@@ -159,6 +161,28 @@ def _env_chain(*names: str, fallback: str | None = None) -> str | None:
+        return
+
+    try:
+        pool = backend.get_pool()
+        pool.close_all()
+    except Exception as exc:  # noqa: BLE001
+        logger.warning("Failed to close superseded content backend pool: {}", exc)
+
+
+def clear_cached_backend() -> None:
+    """Clear and close the currently cached shared content backend."""
+    global _cached_backend, _cached_backend_signature
```

Ruff (0.15.9)

[warning] Using the global statement to update `_cached_backend` is discouraged

(PLW0603)